tsconfig.proxy.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "moduleResolution": "node",
+    "target": "es5",
+    "module": "CommonJS",
+    "lib": ["es2015", "es2016", "es2017", "dom", "ESNext"],
+    "strict": true,
+    "sourceMap": true,
+    "declaration": true,
+    "allowSyntheticDefaultImports": true,
+    "experimentalDecorators": true,
+    "emitDecoratorMetadata": true,
+    "esModuleInterop": true,
+    "outDir": "./proxy/dist/lib",
+    "declarationDir": "./proxy/dist/types",
+    "skipLibCheck": true,
```

Comment:
according to https://www.typescriptlang.org/tsconfig#skipLibCheck we would loose type safety here. Can we drop it?